frontend/common/src/constants/localizedConstants.tsx

```diff
@@ -1140,6 +1141,58 @@ export const getPoolStatus = (
     `Invalid Pool Status '${poolStatusId}'`,
   );
 
+export const poolStream = defineMessages({
+  [PoolStream.BusinessAdvisoryServices]: {
+    id: "PoolStream.BusinessAdvisoryServices",
```

Comment:
May need to check with someone else but I believe we are trying to avoid explicitly setting IDs for intl messages.

Reply:
Resolved with https://github.com/GCTC-NTGC/gc-digital-talent/pull/3805/commits/567f9298bb5f9fab7759fdb7584d26962d089d08